### PR TITLE
Do not reset canTrigger so easily

### PR DESCRIPTION
**Pull request by amiel**
I used this sketch in a Kong Wobbler and found that it frequently triggered dispensing 2 or 3 times per "wobble". The current code waits for it to become stable, but I think that it can read below the threshold at the edges of a wobble, and this only has to happen once to reset the `canTrigger` flag.

With these changes, it must read as stable (below the threshold) for a number of iterations (I've been happy with 5).

### CODE REVIEW

File: Adruino/PetTutorTilt/PetTutorTilt.ino [hunk 1]
```diff
@@ -58,67 +59,74 @@ void setup()
 
   // start with the onboard LED off
   Bean.setLed(0,0,0);
- 
+
```
Comment (amiel) [inline Adruino/PetTutorTilt/PetTutorTilt.ino]: Sorry about the incidental changes. My text editor is configured to remove trailing whitespace. I could undo these changes if you feel strongly about it.

Comment (amiel) [inline Adruino/PetTutorTilt/PetTutorTilt.ino]: You can see a diff in github without the whitespace changes by [appending `?w=1`](https://github.com/PetTutor/PetTutor-SDK-Demos/pull/1/files?w=1)

**Comment by smartanimaltraining** [comment]
no action.  Bean external BLE no longer made
